incremental-font-transfer/src/ift_extend.rs

```diff
@@ -0,0 +1,83 @@
+//! IFT Extension
+//!
+//! This command line tool executes the IFT extension algorithm (<https://w3c.github.io/IFT/Overview.html#extend-font-subset>) on an IFT font.
+
+use std::collections::HashMap;
+
+use clap::Parser;
+use incremental_font_transfer::{
+    patch_group::{PatchGroup, UriStatus},
+    patchmap::SubsetDefinition,
+};
+use read_fonts::collections::IntSet;
+use skrifa::FontRef;
+
+#[derive(Parser, Debug)]
+#[command(version, about = "Run the IFT extension algorithm (https://w3c.github.io/IFT/Overview.html#extend-font-subset) on an IFT font.", long_about = None)]
```

Comment:
do you need to specify that `long_about = None`? I'd expect that to be the default behaviour.